### PR TITLE
global package.json

### PR DESCRIPTION
**Pull request by Kouprin**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[submodule "eth2near/ethashproof"]
-	path = eth2near/ethashproof
+[submodule "ethashproof"]
+	path = ethashproof
```
Comment (frol) [inline .gitmodules:2]: Should we move it back into `eth2near`? :smile:

File: DEV.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Using workspaces: https://classic.yarnpkg.com/en/docs/workspaces/
```
Comment (frol) [inline DEV.md:1]: Let's name this file `CONTRIBUTING.md` following the common practices.

File: write_config.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const { RainbowConfig } = require('rainbow-bridge-utils')
```
Comment (frol) [inline write_config.js:1]: This file is listed in .gitignore. Did you mean to still push it?

**Review by frol — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (Kouprin) [reply .gitmodules]: ++

File: DEV.md (hunk 2 above)
Comment (Kouprin) [reply DEV.md]: ++

File: write_config.js (hunk 3 above)
Comment (Kouprin) [reply write_config.js]: ++

File: cli/commands/status.js [hunk 4]
```diff
@@ -339,8 +339,8 @@ class ServicesStatus {
   }
 
   processArgs (args) {
-    var res = []
-    for (var i = 0; i + 1 < args.length; i++) {
+    const res = []
+    for (let i = 0; i + 1 < args.length; i++) {
       if (args[i].startsWith('--')) {
         res.push(args[i].substring(2, args[i].length) + '=' + args[i + 1])
       }
```
Comment (Kouprin) [inline cli/commands/status.js:346]: is it okay to JS? const res; res.push(data)? It seems linter fixed this one automatically.

Comment (frol) [reply cli/commands/status.js:346]: It is valid JS... it is fine... `const` is only prevents the reassignment of the variable, but not the mutation of the object itself :laughing:

Comment (Kouprin) [reply cli/commands/status.js:346]: Not sure is it constant array or constant address of array.

**Review by frol — APPROVED** [review]

File: cli/commands/status.js (hunk 4 above)
Comment (frol) [reply cli/commands/status.js:346]: It is constant address of array

Comment (Kouprin) [reply cli/commands/status.js:346]: Resolved.

**Pull request merged**